README.md

```diff
@@ -44,10 +44,15 @@ Our other repositories are:
 
 ### Common steps
 
+On MacOS, using [Homebrew](https://brew.sh/) for package management is highly recommended. This helps avoid some known installation issues.
+
 1. Install pre-requisites for setup:
     * [jq](https://stedolan.github.io/jq/): `brew install jq`
     * [terraform](https://www.terraform.io/): `brew install terraform` or `brew install tfenv` and use `tfenv` to install `terraform ~> 1.4.0`
     * [cf-cli@8](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html): `brew install cloudfoundry/tap/cf-cli@8`
+    * [postgresql](https://www.postgresql.org/): `brew install postgresql@15`
```

Comment:
[question] cloud.gov RDS is still postgresql@12, should we direct people to install that? I don't know of any incompatibilities between the versions so it probably doesnt matter